test/e2e/space_test.go

```diff
@@ -128,29 +125,118 @@ func TestSpace(t *testing.T) {
 			}
 
 			// when
-			err := hostAwait.Client.Create(context.TODO(), space)
+			err := hostAwait.Client.Create(context.TODO(), s)
 
 			// then
 			require.NoError(t, err)
-			space, err = hostAwait.WaitForSpace(space.Name, wait.UntilSpaceHasConditions(ProvisioningFailed("unknown target member cluster 'unknown'")))
+			s, err = hostAwait.WaitForSpace(s.Name, wait.UntilSpaceHasConditions(ProvisioningFailed("unknown target member cluster 'unknown'")))
 			require.NoError(t, err)
 
-			t.Run("delete space", func(t *testing.T) {
-				// now, delete the Space and expect that the NSTemplateSet will be deleted as well,
-				// along with its associated namespace
-
+			t.Run("unable to delete space", func(t *testing.T) {
 				// when
-				err = hostAwait.Client.Delete(context.TODO(), space)
+				err = hostAwait.Client.Delete(context.TODO(), s)
 
-				// then
+				// then it should fail while the member cluster is unknown (ie, unreachable)
 				require.NoError(t, err)
-				_, err = hostAwait.WaitForSpace(space.Name, wait.UntilSpaceHasConditions(TerminatingFailed("Cannot delete NSTemplateSet: unknown target member cluster: 'unknown'")))
+				s, err = hostAwait.WaitForSpace(s.Name, wait.UntilSpaceHasConditions(TerminatingFailed("cannot delete NSTemplateSet: unknown target member cluster: 'unknown'")))
 				require.NoError(t, err)
+
+				t.Run("update target cluster to unblock deletion", func(t *testing.T) {
+					// given
+					s.Spec.TargetCluster = memberAwait.ClusterName
+					// when
+					err = hostAwait.Client.Update(context.TODO(), s)
+
+					// then it should fail while the member cluster is unknown (ie, unreachable)
+					require.NoError(t, err)
+
+					t.Run("space should be finally deleted", func(t *testing.T) {
+						// when
+						err = hostAwait.WaitUntilSpaceDeleted(s.Name)
+						// then
+						require.NoError(t, err)
+					})
+				})
 			})
 		})
 	})
 }
 
+func TestUpdateSpace(t *testing.T) {
```

Comment:
```suggestion
func TestUpdateSpaceTier(t *testing.T) {
```
Would it be better to rename this func to be specifically about updating the tier? Also, maybe we should put this test in the nstemplatetier_test.go file?

Reply:
In an upcoming PR this test will also contain a series of subtests for when the Space is migrated to another member cluster, so I'd rather keep it named as it is now